kumascript/macros/JsSidebar.ejs

```diff
@@ -137,36 +137,36 @@ var text = mdn.localStringMap({
     'Guide_keyed_collections': 'Collections avec clés',
     'Guide_Objects': 'Manipuler les objets',
     'Guide_OOP': 'Détails du modèle objet',
-    'Guide_promises' : 'Using promises',
+    'Guide_promises' : 'Utiliser les promesses',
     'Guide_Iterators_Generators': 'Itérateurs et générateurs',
-    'Guide_Meta': 'Meta-programmation',
-    'Guide_Modules': 'JavaScript modules',
+    'Guide_Meta': 'Métaprogrammation',
+    'Guide_Modules': 'Modules JavaScript',
     'Complete_beginners': 'Débutant',
     'Basics': 'Les bases en JavaScript',
     'First_steps': 'Premiers pas en JavaScript',
     'Building_blocks': 'Les briques JavaScript',
     'Intermediate': 'Intermédiaire',
-    'Introducing_objects': 'Introducing JavaScript objects',
-    'Client-side_APIs': 'Client-side web APIs',
-    'Frameworks': 'Client-side JavaScript frameworks',
+    'Introducing_objects': 'Introduction aux objets JavaScript',
+    'Client-side_APIs': 'Les API Web côté client',
+    'Frameworks': 'Les frameworks JavaScript côté client',
     'Re-introduction': 'Réintroduction à JavaScript',
     'Data_structures': 'Structures de données en JavaScript',
     'Equality': 'Différents tests d\'égalité',
-    'Closures': 'Closures',
+    'Closures': 'Fermetures (closures)',
     'Advanced': 'Avancé',
-    'Inheritance': 'Héritage et la chaîne de prototypes',
+    'Inheritance': 'L\'héritage et la chaîne de prototypes',
     'Strict_mode': 'Mode strict',
     'Typed_arrays': 'Tableaux typés en JavaScript',
     'Memory_Management': 'Gestion de la mémoire',
     'Event_Loop': 'Concurrence et boucle des événements',
-    'Reference': 'Références\xa0:',
+    'Reference': 'Références&nbsp;:',
     'Global_Objects': 'Objets natifs',
     'Operators': 'Expressions & opérateurs',
     'Statements': 'Instructions & déclarations',
     'Functions': 'Fonctions',
     'Classes': 'Classes',
-    'Errors': 'Errors',
-    'More': 'Plus',
+    'Errors': 'Erreurs',
+    'More': 'Divers',
     'Lexical_grammar': 'Grammaire lexicale',
     'Enumerability':'Rattachement des propriétés',
```

Comment:
And at line 175 (I can't suggest :/), maybe `'Template_strings': 'Gabarit de chaînes de caractères',` instead?

```suggestion
    'Enumerability': 'Rattachement des propriétés',
```